.github/workflows/build-prod.yml

```diff
@@ -0,0 +1,34 @@
+name: Build/Test
```

Comment:
If the build and unit tests pass on dev and only dev merges into main, isn't re-running the same build/test steps on main redundant?

Reply:
it wouldn't hurt, instead make sure everything cool on main branch.